src/services/http/http.ts

```diff
@@ -36,6 +37,27 @@ export const http = got.extend({
     http: new HttpAgent({ keepAlive: true }),
     https: new HttpsAgent({ keepAlive: true }),
   },
+  retry: {
+    limit: 10,
+    methods: ["GET", "PUT", "HEAD", "DELETE", "OPTIONS", "TRACE"],
+    statusCodes: [408, 413, 429, 500, 502, 503, 504, 521, 522, 524],
+    errorCodes: [
+      "ETIMEDOUT",
+      "ECONNRESET",
+      "EADDRINUSE",
+      "ECONNREFUSED",
+      "EPIPE",
+      "ENOTFOUND",
+      "ENETUNREACH",
+      "EAI_AGAIN",
+      "EADDRNOTAVAIL",
+      "EHOSTUNREACH",
+    ],
+    maxRetryAfter: undefined,
+    calculateDelay: ({ computedValue }) => computedValue,
+    backoffLimit: ms("5s"),
+    noise: 100,
+  },
```

Comment:
This is based on the defaults documented here: https://github.com/sindresorhus/got/blob/3822412385506a1efef6580d270eae14086b9b43/documentation/7-retry.md#retry